src/go/configgenerator/listener_generator_test.go

```diff
@@ -962,13 +962,13 @@ func TestPathMatcherFilter(t *testing.T) {
 						{
 							Selector: "1.cloudesf_testing_cloud_goog.Foo",
 							Pattern: &annotationspb.HttpRule_Get{
-								Get: "foo/{foo_bar}",
```

Comment:
Is this a breaking change? We allowed HttpRule without a prefix `/`, but now we enforce it?

Reply:
Starting with '/' is required. 

https://cloud.google.com/endpoints/docs/grpc-service-config/reference/rpc/google.api#path-template-syntax